src/extlib/src/smpeg-2.0.0/configure.in

```diff
@@ -61,6 +61,11 @@ AC_PROG_INSTALL
 dnl Work around the lack of certain typedefs.
 AC_TYPE_SOCKLEN_T
 
+dnl The script previously overwrote CXXFLAGS with CFLAGS, but that's not always okay
+dnl Save CFLAGS here, then clear it, and finally use it to append to both
+save_CFLAGS="$CFLAGS"
+CFLAGS=""
+
```

Comment:
Just curious, is this also upstream or when we update smpeg will we need to re-add these changes on top again?